src/core/storage/secureStorage/secureStorage.ts

```diff
@@ -6,9 +6,9 @@ import {
 enum KeyStoreKeys {
   APP_PASSCODE = "app-login-passcode",
   APP_OP_PASSWORD = "app-operations-password",
-  APP_PASSWORD_SKIPPED = "app-password-skipped",
   SIGNIFY_BRAN = "signify-bran",
   MEERKAT_SEED = "app-meerkat-seed",
+  RECOVERY_WALLET = "recovery-wallet",
```

Comment:
What is this used for and why is it in the secure storage? @Sotatek-DukeVu

Reply:
Recovery wallet option and create wallet option have same flow except generate/verify seed phrase screen. 

If user reload app when generate/verify seed phrase, I need this flag to know what option user chosen and nav to the correct screen.

Reply:
And When I did this task, I think secure storage is less security because it stored value in `localStorage` on browser

Reply:
OK, this needs to move to Basic storage just like `APP_PASSWORD_SKIPPED` - can you open another PR? cc @sdisalvo-crd

Reply:
Thanks @iFergal I will create a ticket for @Sotatek-DukeVu 👍